src/rqt_publisher/publisher.py

```diff
@@ -51,8 +50,10 @@ def __init__(self, context):
         super(Publisher, self).__init__(context)
         self.setObjectName('Publisher')
 
+        self._node = context._node
```

Comment:
Should it keep a copy of the node or a copy of the context?

Reply:
I'm leaning towards context. Also, shouldn't we check the context type before referencing _node?

Reply:
Saving context makes sense. I don't think a check is necessary. That will be a bug on their end if they set their plugin as a qt_gui plugin and they meant for it to be an rqt_gui plugin.